src/package.json

```diff
@@ -62,7 +62,7 @@
     "url-loader": "^0.6",
     "webpack": "^4.28.3",
     "webpack-bundle-analyzer": "^2.11.1",
-    "webpack-cli": "^2.0.12",
+    "webpack-cli": "^3.2.1",
```

Comment:
Just **remove** webpack-cli instead.  We do not use it anywhere do we?  It's just for make a config file, and we already did that.

Reply:
well, good idea, and I don't know if we use it anywhere. I'll try.

Reply:
I did grep and we don't as far as I can tell...

Reply:
well, without "cli", the webpack compile process bails out:

```
One CLI for webpack must be installed. These are recommended choices, delivered as separate packages:
 - webpack-cli (https://github.com/webpack/webpack-cli)
   The original webpack full-featured CLI.
```
I'll try to figure out if there is a recommended setup.

Reply:
OK, I didn't know it was implicit.  OK.